src/store/app.ts

```diff
@@ -5,11 +5,23 @@ import Keycloak from 'keycloak-js'
 import { defineStore } from 'pinia'
 import silentCheckSsoHtmlUrl from '@/assets/silent-check-sso.html?url'
 
+/**
+ * The user roles of the Keycloak realm
+ * that are relevant to the frontend.
+ */
+export enum UserRole {
+    Buyer = 'Buyer',
+    Employee = 'Employee',
+    SiteAdmin = 'Administrator',
```

Comment:
the role in only named admin

Reply:
But the Keycloak role name is "site-admin". Btw, in our domain model it says "Customer", "Employee", and "Admin". Maybe we should decide on one set of names.

Reply:
I settled on: The frontend defines the roles from the domain model and has a lookup table for mapping the user roles to the role names coming from Keycloak.

Reply:
well, but the domain model roals will mirror keycloak roles as soon as I fix the domain model (today or tomorrow)
I would really recommend already using the keycloak names